canvasapi/course.py

```diff
@@ -1698,6 +1698,33 @@ def get_quiz(self, quiz):
 
         return Quiz(self._requester, quiz_json)
 
+    def get_quiz_overrides(self, **kwargs):
+        """
+        Retrieve the actual due-at, unlock-at, \
+        and available-at dates for quizzes based on \
+        the assignment overrides active for the current API user.
+
+        :calls: `GET /api/v1/courses/:course_id/quizzes/assignment_overrides \
+        <https://canvas.instructure.com/doc/api/quiz_assignment_overrides.html#method.quizzes/quiz_assignment_overrides.index>`_
+
+        :param quiz_assignment_overrides: An array of quiz IDs. If omitted, overrides \
```

Comment:
There's no need to explicitly document the param `quiz_assignment_overrides`. We usually only document required params (those that actually appear in the method signature).